index.html

```diff
@@ -1,135 +1,43 @@
 ---
 layout: default
+title:
 ---
-
-<!-- Banner -->
-<section id="banner">
-    <div class="content">
-        <header>
-            <h2>Wild Tree Tech</h2>
-            <p>Data Done Better.</p>
-        </header>
-        <span class="image"><img src="images/logo.jpg" alt="" /></span>
+<div class="primary-banner main-page">
+  <div class="container">
+    <div class="row justify-content-center">
+      <div class="col-lg-10 col-md-12">
+        <h2>Data Done Better</h2>
+        <p>Building data driven products for our clients.</p>
+    	</div>
     </div>
-    <a href="#science" class="goto-next scrolly">Next</a>
-</section>
-
-<!-- Data science -->
-<section id="science" class="spotlight style1 bottom">
-    <span class="image fit main"><img src="images/more-woah.jpg" alt="" /></span>
-    <div class="content">
-        <div class="container">
-            <div class="row">
-                <div class="4u 12u$(medium)">
-                    <header>
-                        <h2>Welcome!</h2>
-                        <p>Wild Tree Tech - the data engineering experts.</p>
-                    </header>
-                </div>
-                <div class="8u$ 12u$(medium)">
-                    <p>
-Wild Tree Tech is a small company focused on improving the user experience of your data, what we like to call data experience.
-It is operated by Tim Head, a former <a href="https://home.cern/">CERN</a> physicist who is
-known in the Python community as maintainer of <a href="https://scikit-optimize.github.io/">scikit-optimize</a>
-and <a href="https://mybinder.org/">binder</a>. Wild Tree Tech offers training courses to individuals and corporate clients, as well as data experience
-consulting services.
-</p>
-<p>
-                        Getting started with AI? Launching a new data science product? Got data that is hard to use? Need Help with an existing project?<br />
-                        Together we can make it happen. <a href="/index.html#contact">Let's talk!</a>
-                    </p>
-                </div>
-            </div>
+  </div>
+</div>
+
+<div class="container index" id="main-section">
+  <div class="row justify-content-center">
+    <div class="col-lg-10 col-md-12">
+      <div class="row justify-content-center content-section separated">
+        <div class="col-md-12">
+          <p>Our team of technology experts builds ambitious solutions for clients all around the world, from startups to UN organisations. We specialise in building digital products that leverage machine-learning and AI. Find out more over cup of coffee or tea.</p>
```

Comment:
"Find out more over cup of coffee or tea" -> "Find out more over A cup of coffee or tea."